src/ResourceManager/HDInsight/Commands.HDInsight.Test/UnitTests/ConfigurationTests.cs

```diff
@@ -51,7 +51,7 @@ public void CanCreateNewConfig()
                     f.WriteObject(
                         It.Is<AzureHDInsightConfig>(
                             c =>
-                                c.ClusterType == HDInsightClusterType.Hadoop &&
+                                c.ClusterType == "Hadoop" &&
```

Comment:
There really isn't a need for a constant here. These are tests and are subject to change as we feel given the coverage we want for the code. Definitely doesn't need to be case insensitive.